src/redux/books/books.js

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+
+const Books = () => {
+  const BooksData = {
+    gender: 'Action',
+    title: 'The Hunger Games',
+    director: 'Suzanne Collins',
+    porcentage: '64%',
+    chapter: 'Chapter 17',
+  };
+
+  const Book = () => {
```

Comment:
- [ ] Kindly extract this **`Book`** component into a new component under `src/component`. Remember to separate the form from the Book component as, technically, they are independent of each other.
- [ ] Your **`Book`** component should also take in the book data as a prop and render those data